src/main/java/com/mongodb/hibernate/internal/translate/AbstractMqlTranslator.java

```diff
@@ -201,6 +212,10 @@ abstract class AbstractMqlTranslator<T extends JdbcOperation> implements SqlAstT
 
     private final Set<String> affectedTableNames = new HashSet<>();
 
+    private @Nullable Limit limit;
+    private @Nullable JdbcParameter firstRowParameter;
+    private @Nullable JdbcParameter maxRowsParameter;
```

Comment:
Compared with `AbstractSqlAstTranslator` counterpart below:
```
private Limit limit;
private JdbcParameter offsetParameter;
private JdbcParameter limitParameter;
```
I renamed the two `JdbcParameter`s to avoid confusion (particularly the `limitParameter`). The new variable names just correspond to the `Limit` field names.